core/codegen/tests/compile-test.rs

```diff
@@ -85,6 +85,7 @@ fn run_mode(mode: &'static str, path: &'static str) {
     config.clean_rmeta();
 
     config.target_rustcflags = Some([
+        "--edition=2018".into(),
```

Comment:
Do we actually need this? This is more of a curiosity level question than a "change this" comment.

Reply:
No, it's not needed anymore. This might have been because `rocket_codegen` was itself a 2015 edition crate when I started working on this, and edition hygiene is a bit wonky sometimes. I'll make sure we have appropriate test coverage.